features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.types.listing/listing.js

```diff
@@ -1,3 +1,42 @@
 function onRequest(context){
+    var log = new Log("listing.js");
+    var DTYPE_CONF_DEVICE_TYPE_KEY = "deviceType";
+    var DTYPE_CONF_DEVICE_TYPE_LABEL_KEY = "label";
+    var DTYPE_UNIT_NAME_PREFIX = "cdmf.unit.device.type.";
+    var DTYPE_UNIT_NAME_SUFFIX = ".type-view";
+    var DTYPE_UNIT_CONFIG_PATH = "/private/conf/device-type.json";
+    var DTYPE_UNIT_LISTING_TEMPLATE_PATH = "/public/templates/listing.hbs";
 
+    var viewModel = {};
+    var deviceModule = require("/app/modules/device.js").deviceModule;
+    var data = deviceModule.getDeviceTypes();
+
+    if(data.data){
```

Comment:
This code is moved to components and fixed in [#e537e2d](https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1)